x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -258,27 +256,73 @@ function _generateMappings(
             dynProperties = histogram(field);
             matchingType = field.object_type_mapping_type ?? '*';
             break;
+          case 'ip':
```

Comment:
There are some types that are not specified in the `object_type` field definition in package-spec: ip, wildcard, group or match_only_text

Should these be added too here?